cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,18 @@
+const { generateUser } = require('../support/generate');
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const {username, email, password} = generateUser();
 
+cy.visit('https://react-redux.realworld.io')
+cy.get(':nth-child(3) > .nav-link').click()
```

Comment:
it's good to check also if the button contains what you need (e.g. 'Sign up' text)